package.json

```diff
@@ -78,7 +78,6 @@
   "devDependencies": {
     "@grpc/proto-loader": "^0.5.1",
     "@types/concat-stream": "^1.6.0",
-    "@types/extend": "^3.0.0",
```

Comment:
I'm surprised this works - we are using the actual `extend` package in many other places.  The TypeScript types at `@types/extend` should have no material impact on the code at runtime.  I think we probably want to add this back?